db/seeds.d/03-permissions.rb

```diff
@@ -1,169 +1,3 @@
-permissions = [
```

Comment:
I needed to do something similar for templates and I ended up moving it to separate file in [seeds](https://github.com/theforeman/foreman/pull/4320/files#diff-f63d6f59b7e054afbab817ac932cd46bR15), I suppose we should do it the same way. I'm happy to change my approach if you think it makes more sense to put it under app/services/foreman/constants but I was thinking seed related data should be in db/seeds.d. I'd appreciate others ideas.